components/ActionSheet/tests/unit/MDCActionSheetTest.m

```diff
@@ -204,4 +218,142 @@ - (void)testSetNilTitleAndMessageColor {
                         [UIColor.blackColor colorWithAlphaComponent:0.6f]);
 }
 
+#pragma mark - Opening height
+
+- (void)addActions:(NSUInteger)actions {
+  for (NSUInteger i = 0; i < actions; ++i) {
+    MDCActionSheetAction *action = [MDCActionSheetAction actionWithTitle:@"Title"
+                                                                   image:nil
+                                                                 handler:nil];
```

Comment:
Tip: Consider including the action index in the title: `[NSString stringWithFormat:@"Action #%@", @(i)];`

Reply:
Done